komodo/release_transpiler.py

```diff
@@ -4,13 +4,15 @@
 import itertools
 import os
 
+import yaml
+
 from komodo.matrix import format_release, get_matrix
 from komodo.prettier import load_yaml, write_to_file
 
 
-def build_matrix_file(release_base, release_folder, builtins):
+def build_matrix_file(release_base, release_folder, builtins, coords):
     files = {}
-    py_keys = [py_ver for _, py_ver in get_matrix()]
+    py_keys = [f"py{py_version.replace('.', '')}" for py_version in coords["py"]]
```

Comment:
My thought was that by using the `coords` dict, I think it should be possible to get rid of `py_ver` and `rhel_ver` completely. Instread of passing those variable around, pass the `coords` dict around. Maybe I'm missing something and it's not possible, but it seems like the `coords` dict should be able to have entries like `macos_ver:['10.15', '10.16']` and the code will just accommodate it as another constraint to add to the release string. This would mean looping over both the keys and the values to generate all the various release name strings.

(I _think_ this would mean we could also do things like this with another entry in the `coords` dict > https://github.com/equinor/komodo/pull/331 )

Reply:
I fully understand your point.
In order to keep the scope of the issue, I would prefer to use another issue i.e. #332 to consider turning komodo agnostic to os and dev system.

Reply:
Right -- I think this point is out of date wrt the code, but either way, I will just focus on #332 for this sort of thing.